<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>用户注册项目</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密  码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRegname">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>admin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRepass">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>123456</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbegister">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>86</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
  <widget class="QListView" name="lstUser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>491</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pbFind">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户信息展示</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
